<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-HN641TBH" ignored="0" name="InnerRoots" tests="5" failures="1" timestamp="2020-08-26T17:48:11 CDT" time="0.075" errors="0">
  <testcase name="testShell" time="0.006" classname="test.Root3"/>
  <testcase name="skippingTests" time="0.008" classname="test.Root4">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.Root4.skippingTests(Root4.java:17)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- skippingTests -->
  <testcase name="day3" time="0.002" classname="test.Root3"/>
  <testcase name="day3" time="0.001" classname="test.Root3"/>
  <testcase name="day3" time="0.003" classname="test.Root3"/>
</testsuite> <!-- InnerRoots -->
